### PR TITLE
Update advice on installing conda-build

### PR DESCRIPTION
**Pull request by kathatherine**
### Description

Closes https://github.com/conda/conda-build/issues/5212.

### Checklist - did you ...

- ~[ ] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?~
     
- ~[ ] Add / update necessary tests?~
- [X] Add / update outdated documentation?

### CODE REVIEW

File: docs/source/user-guide/getting-started.rst [hunk 1]
```diff
@@ -16,16 +16,17 @@ Prerequisites
...
-- `Miniconda or Anaconda <https://docs.anaconda.com/anaconda/install/>`_
+- `Miniconda <https://docs.anaconda.com/free/miniconda/>`_ or `Anaconda <https://docs.anaconda.com/free/anaconda/install/>`_
 - conda-build
 - Git
 
 The most straightforward way to do this is to install Miniconda or
 Anaconda, which contain conda, and then use conda to install conda-build
-and Git. Make sure you install these packages into a new environment
-and not your base environment.::
+and Git. Make sure you install these packages into your base environment.::
```
Comment (kathatherine) [inline docs/source/user-guide/getting-started.rst:25]: I assume one would have to still install Git into base, in order to have everything needed to follow the tutorial, yes?

File: docs/source/user-guide/getting-started.rst [hunk 2]
```diff
@@ -16,16 +16,17 @@ Prerequisites
...
 
-    conda create -n my-conda-build-environment conda-build git
+    conda create -n base conda-build git
```
Comment (beeankha) [inline docs/source/user-guide/getting-started.rst:27]: If indeed `git` needs to be installed into the base env, then should this command be `conda install -n base conda-build git` instead?

Comment (kathatherine) [reply docs/source/user-guide/getting-started.rst:27]: Oh, you're absolutely right. The environment doesn't need to be created anymore. Whoops!

**Review by beeankha — APPROVED** [review]

File: docs/source/user-guide/getting-started.rst (hunk 1 above)
Comment (jezdez) [reply docs/source/user-guide/getting-started.rst:25]: I would keep it, yeah.

**Review by jezdez — APPROVED** [review]

**Pull request merged**